### ISSUE TITLE
[SCREENREADER] While typing into input, after each character an error is read

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#5875]
## Description

While typing numbers into the input, after each character is read by the screen reader an error is read. 

For example, entering 04101 as Zip Code: 
> “0 Alert Error Zip Code is required Zip Code must be five digits 4  Alert Error Zip Code is required Zip Code must be five digits 1  Alert Error Zip Code is required Zip Code must be five digits 0 Alert Error Zip Code is required Zip Code must be five digits 1 Alert Error Zip Code is required Zip Code must be five digits ” 

Once all five digits are entered, the error message repeats again, although the content entered is valid. This may be a bug in input validation, as this has been found on Facility Locator https://github.com/department-of-veterans-affairs/va.gov-team/issues/529 and Authenticated Experience Address Validation https://github.com/department-of-veterans-affairs/va.gov-team/issues/5323.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to enter values in the input and not experience an error notification until my input is complete.

## Environment

* Operating System: Windows, Mac
* Browser: IE11, Chrome
* Screenreading device: JAWS, also visible without screen reader
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/profile/` in browser
2. Start screenreading device listed in Environment
3. Navigate to edit the Mailing address
4. Navigate to the Zip Code input
5. Delete the value and begin typing a new Zip Code
6. Verify the validation begins while the focus is on the input and only a few characters entered

## Possible Fixes (optional)

* Consider using an `<input type="number"…` and include pattern attributes for HTML5 validation

## WCAG or Vendor Guidance (optional)

* [Error Identification: Understanding SC 3.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)

## Screenshots or Trace Logs

![error-validation-bug](https://user-images.githubusercontent.com/57469/74484710-1bab8b00-4e87-11ea-94e7-b7240fb7b6e5.png)

**Comment by jenstrickland** [comment]
This appears to be a bug in the form input field validation, and a similar issue was found in https://github.com/department-of-veterans-affairs/va.gov-team/issues/529

**Comment by alexpappasoddball** [comment]
@brandonrapp Throwing your way for prioritization

**Comment by taylorkaren** [comment]
Forms Library Team Ticket: https://app.zenhub.com/workspaces/forms-library---platform-spike-team-61b0ae1f2cd3c30014e8a5b0/issues/department-of-veterans-affairs/formulate/88.

**Comment by ken-non** [comment]
@k80bowman - who would be an accessibility SME we could run this by when we're ready for UAT?

**Comment by k80bowman** [comment]
@ken-non I saw in Slack that Shira recommended going through the Collaboration Cycle and that might be the best way to go here.

**Comment by CourtneyChambers77** [comment]
when entering the values in the input the experience an error notification when deleting a character or number completely from the  zip code city and state. The behavior should be a user should be able to delete or erase characters when making a mistake or incorrect data. this should happen when the selects submit button 
![image.png](https://images.zenhubusercontent.com/61b2260296ae8721195bcef1/de07fc2d-6443-4ef3-9595-d793fd93f2ec)

**Comment by taylorkaren** [comment]
Thanks for the retest, Courtney. I have updated the corresponding ticket in the FLT Repo Accessibility backlog pipeline.
